src/modules/UI/scenes/ChangeMiningFee/ChangeMiningFee.ui.js

```diff
@@ -0,0 +1,68 @@
+import React, {Component} from 'react'
+import {View, Text, TextInput} from 'react-native'
+
+import RadioButton from './components/RadioButton.ui'
+import * as FEE from '../../../../constants/FeeConstants'
+
+import style from './style'
+
+const feeOptions = [
+  { value: FEE.LOW_FEE, label: 'Low' },
+  { value: FEE.STANDARD_FEE, label: 'Standart' },
+  { value: FEE.HIGH_FEE, label: 'High' },
+  { value: FEE.CUSTOM_FEE, label: 'Custom (Advanced)' },
+]
+
+export default class ChangeMiningFee extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      fee: props.fee,
+      feeSetting: props.feeSetting,
+    }
+  }
+
+  componentWillUnmount () {
+    if (this.state.fee !== this.props.fee || this.state.feeSetting !== this.props.feeSetting) {
+      this.props.onSubmit(this.state.feeSetting, this.state.fee)
+    }
+  }
+
+  hendlePress = (feeSetting) => this.setState({ feeSetting, fee: '' });
+  hendleChange = (fee) => this.setState({ fee });
+
+  render () {
+    const { feeSetting, fee } = this.state
+
+    return (
+      <View style={style.wrapper}>
+        <Text style={style.header} >
+          Warning: Low Fees may cause long delays in transaction confirmation
```

Comment:
When displaying text, please use 
`import strings from '...locales/default'`
...
`strings.enUS[ {{string_key}} ]`

If the desired text does not exist in the strings file, please create it